india_compliance/gst_india/api_classes/nic/e_waybill.py

```
@@ -0,0 +1,195 @@
+            "username": self.username,
+            "password": self.password,
+            "app_key": self.app_key,
+        }
+
+        return self._make_request("POST", endpoint="auth", json=json_data)
+
+    @change_base_path("standard/ewb/master")
+    def get_transporter_details(self, transporter_id):
+        return super().get_transporter_details(transporter_id)
+
+    def is_ignored_error(self, response_json):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)